src/sentry/deletions/defaults/project.py

```diff
@@ -54,19 +54,11 @@ def get_child_relations(self, instance):
                 for m in model_list
             ]
         )
-
-        model_list = (models.GroupMeta, models.GroupResolution, models.GroupSnooze)
```

Comment:
Why do we remove these? Just already handled in the direct group deletion?

Reply:
Yes. Project now has Group as a child_relation and Group has these models as child_relations.

This was likely a workaround to be able to bulk delete across multiple groups. That isn't necessary anymore as I've changed bulk deletions to support `__in` lookups as well.